src/api/resourceApi.ts

```diff
@@ -183,11 +195,23 @@ export const deleteResourceContent = (resourceIdentifier: string, contentIdentif
   });
 };
 
-export const getResourceContents = (identifier: string): Promise<AxiosResponse<Content[]>> => {
-  return authenticatedApiRequest({
+export const getResourceContents = async (identifier: string): Promise<ResourceContents> => {
+  const contentResponse = await authenticatedApiRequest({
     url: encodeURI(`${API_PATHS.guiBackendResourcesPath}/resources/${identifier}/contents`),
     method: 'GET',
   });
+  const resourceContent: ResourceContents = {
+    masterContent: { identifier: '', features: { dlr_content_title: '', dlr_content: '' } },
+    sideContent: [],
+  };
+  contentResponse.data.forEach((content: Content) => {
+    if (content.features.dlr_content_master === 'true') {
+      resourceContent.masterContent = content;
+    } else {
+      resourceContent.sideContent.push(content);
+    }
+  });
+  return resourceContent;
```

Comment:
kanskje vi skulle lagt inn filteret her også - for å gjemme filer vi ikke bryr oss om ? som metadata_external.json. Såvidt jeg skjønner så blir den bare fjernet ved upload av nye ressurser?

Reply:
Vet vi at vi ikke ønsker metadata_external.json overhodet? Inneholder ikke den informasjon om når man besøkte sida f.eks?

Reply:
utifra det som ble sagt på slack så er det ingen grunn til å vise den - så vi kan like gjerne filtere den ut ved uthenting av content